src/_tests/example_layouts/__snapshots__/_wireframe.test.tsx-nested-wireframe.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
@orta How's this? I think before you were keeping the background colour. What do you think is best when background colours are included?

Reply:
This is 👍  by me - at some point I'd like to do more work on making it look pretty, but keeping it the same is

Reply:
Did you want to keep the background/border colours?

Reply:
If it's blank - yeah 👍